src/Infrastructure/LeanCode.AuditLogs/README.md

```diff
@@ -0,0 +1,151 @@
+# LeanCode.AuditLogs
```

Comment:
And now, the big question - should it be here, or should it be in the docs?

Reply:
I believe it should be in the docs, as it is something that describes how to configure a feature that this package provides. On the other hand this is strictly connected to this package and it's references, so here is a good place for it as well.
@wojtek2288 do you have any thoughts on this?

Reply:
I also think this should be in docs - we don't yet have a sensible approach to READMEs, so let's keep everything in a single/correct place.